tools/gfx-unit-test/shader-cache-tests.cpp

```diff
@@ -817,13 +852,28 @@ namespace gfx_test
             submitGPUWork();
             generateNewPipelineState(2);
             submitGPUWork();
+            getCacheFile(test4Lines);
 
-            // Cache limit 3, access shaders A then B then C
             device->queryInterface(SLANG_UUID_IShaderCacheStatistics, (void**)shaderCacheStats.writeRef());
             SLANG_CHECK(shaderCacheStats->getCacheMissCount() == 1);
             SLANG_CHECK(shaderCacheStats->getCacheHitCount() == 2);
             SLANG_CHECK(shaderCacheStats->getCacheEntryDirtyCount() == 0);
-            getCacheFile(test4Lines);
+
+            SLANG_CHECK(test4Lines.getCount() - 1 == 3);
```

Comment:
-1 here is confusing. Don't append empty lines in `getCacheFile`.

Reply:
Fixed